SU2_CFD/src/solvers/CDiscAdjSolver.cpp

```diff
@@ -362,25 +362,25 @@ void CDiscAdjSolver::ExtractAdjoint_Solution(CGeometry *geometry, CConfig *confi
   END_SU2_OMP_MASTER
 
   /*--- Extract and store the adjoint of the primal solution at time n ---*/
-  if (time_n_needed) {
+  if (time_n_needed && !CrossTerm) {
```

Comment:
As discussed in #1260 . I tested my cht case which gave identical residuals so I would expect it is the same for fsi, but as there is a testcase there is no guessing necessary :)

Reply:
Well, ... this commit lived a short life of 30 minutes